api/HTMLAnchorElement.json

```diff
@@ -412,10 +412,10 @@
               "version_added": "41"
             },
             "safari": {
-              "version_added": "11.1"
+              "version_added": "14"
```

Comment:
Added in https://trac.webkit.org/changeset/257707/webkit, 11.1 can't have been right.